### PR TITLE
Add numeric display option for Heat and Ores

### PR DESCRIPTION
**Pull request by AnodeCathode**
For modded TFC with Metals having melt temps > 3000 degrees, it's become clear that the Temp Enum is insufficient. So this adds an advanced option to the ore display that also applies to the Heat items and displays the specific numeric amounts.

### CODE REVIEW

File: src/main/java/net/dries007/tfc/objects/items/metal/ItemOreTFC.java [hunk 1]
```diff
@@ -193,6 +208,15 @@ public boolean canMelt(ItemStack stack)
         return ore.canMelt();
     }
 
+    @Override
+    public float getMeltTemp(ItemStack stack)
+    {
+       if (this.canMelt(stack)){
```
Comment (DisasterMoo) [inline src/main/java/net/dries007/tfc/objects/items/metal/ItemOreTFC.java]: Minor code style issue

File: src/main/resources/assets/tfc/lang/en_us.lang [hunk 2]
```diff
@@ -87,6 +87,7 @@ tfc.tooltip.oredictionaryentry=§fOre Dictionary name:§7 %s
 tfc.tooltip.oredictionaryentries=§fOre Dictionary names:
 tfc.tooltip.metal=§fMetal:§7 %s
 tfc.tooltip.units=%d units
+tfc.tooltip.melttemp=%d°
```
Comment (DisasterMoo) [inline src/main/resources/assets/tfc/lang/en_us.lang]: This opens up debate of which units we use, which for me seems to be Celsius. If so:
```suggestion
tfc.tooltip.melttemp=%d °C
```

**Review by DisasterMoo — CHANGES_REQUESTED** [review]
Just that minor code style issue, but also let's see what Alc think about the temperature units.

File: src/main/resources/assets/tfc/lang/en_us.lang (hunk 2 above)
Comment (AnodeCathode) [reply src/main/resources/assets/tfc/lang/en_us.lang]: Shouldn't be a space between the value and the symbol. Whether to include the letter for the scale is entirely up to you

Comment (AnodeCathode) [reply src/main/resources/assets/tfc/lang/en_us.lang]: Added without the space but with the C symbol

**Review by DisasterMoo — APPROVED** [review]

**Pull request merged**